### ISSUE TITLE
Rust debugger not working on Windows

### ISSUE THREAD

**Issue by Xoffio** [AstroNvim/astrocommunity#167]
Hi,

I enabled the rust community package `user/plugins/community.lua`:
```lua
return {
  "AstroNvim/astrocommunity",
  { import = "astrocommunity.pack.rust" },
}
```

added `user/plugins/nvim_dap.lua`: 
```lua 
return {
  {
    "mfussenegger/nvim-dap", -- for debbugging
    enabled = true,
  }
}
```

but when I try to run `RustDebuggables` I get the next error "ENOENT: no such file or directory"
![rustdebuggable_error](https://user-images.githubusercontent.com/38369407/234754321-8cddf7fa-8980-4f7a-9418-4267f4395425.jpg)

Something that I noticed is that the rust community package downloads the codelldb in:
```
codelldb_path = "/extension/adapter/codelldb.exe"
liblldb_paht = "/extension/lldb/lib/liblldb.lib"
```

Instead of:
```
codelldb_path = "/codelldb"
liblldb_paht = "/extension/lldb/lib/liblldb"
```

So I manually edited the `rust.lua` file to match the right path in windows. Now nvim is able to run codelldb but then I get this error `ECCONNREFUSED`:
![dap_error_01](https://user-images.githubusercontent.com/38369407/234756039-7c7deb6a-a4f9-45dd-985d-22d935ce8c70.jpg)

Is there anything I can do to make this work?
If so then I can contribute updating the rust pack to check if the system is running windows and if so to point to the right directory.

Thanks in advance!

**Comment by Uzaaft** [comment]
From the docs:
<img width="799" alt="image" src="https://user-images.githubusercontent.com/48220549/235254816-e7f7bb22-e54f-4de0-a25d-407623a2c822.png">
Feel free to add checks to windows if you think they'll work. It might take some time to double check your work.

**Comment by Uzaaft** [comment]
Additional  pics of the docs: 

<img width="865" alt="image" src="https://user-images.githubusercontent.com/48220549/235255817-83d786c9-ec15-4a18-bac7-54b743727aa6.png">

**Comment by Xoffio** [comment]
>  Feel free to add checks to windows if you think they'll work. It might take some time to double check your work.

HI @Uzaaft, can you clarify what you mean by "adding checks" ?   

> Additional pics of the docs:
> 
> <img alt="image" width="865" src="https://user-images.githubusercontent.com/48220549/235255817-83d786c9-ec15-4a18-bac7-54b743727aa6.png">

I actually had to add this to my `user/init.lua` file to make it work. Maybe we can also add it to the rust pack conf file?

**Comment by Uzaaft** [comment]
I was just refering to what you wrote. 

> If so then I can contribute updating the rust pack to check if the system is running windows and if so to point to the right directory.

**Comment by Xoffio** [comment]
@Uzaaft  thank you. I actually read that before. I explained the problem I was having in another thread and someone told me to use the rust pack... then I find out that the rust pack was not going to fix it... Anyway, with the changes I made in #169 the debugger seems to be working on windows.

To make sure that it was working. I deleted the old AstroNvim conf and re-downloaded it. I enabled the rust community pack and enabled the `nvim-dap`. and now it works on windows
